src/test/lib/README.md

```diff
@@ -0,0 +1,11 @@
+# Test library
+
+This contains all files for the test library, which is used by all test binaries (unit tests, benchmarks, fuzzers, gui
+tests)
+
+Generally, the files in this folder should be well separated modules. New code should be added to existing modules or
+(when in doubt) a new module should be created.
+
+The utilities in here are compiled into a library, which does not hold any state. However, the main file `setup_common`
```

Comment:
does this refer to `setup_common.cpp`, `setup_common.h`, or both?

Reply:
both